packages/web-components/fast-foundation/src/avatar/avatar.template.ts

```diff
@@ -0,0 +1,33 @@
+import { html, when } from "@microsoft/fast-element";
+import { Avatar } from "./avatar";
+
+/**
+ * The template for {@link @microsoft/fast-foundation#Avatar} component.
+ * @public
+ */
+export const AvatarTemplate = html<Avatar>`
+    <div
+        class="coin ${x => (x.shape === "square" ? "square" : "circle")}"
```

Comment:
nit - and I could be missing something, but could this be:

```suggestion
    class="coin ${x => x.shape === "square" ? "square" : "circle")}"
```